WordPress/src/main/java/org/wordpress/android/ui/accounts/SignInFragment.java

```diff
@@ -1278,6 +1322,23 @@ public void onAvailabilityChecked(OnAvailabilityChecked event) {
         }
     }
 
+    private void handleUsernameAvailabilityEvent(OnAvailabilityChecked event) {
+        endProgress();
+        if (event.type != IsAvailable.USERNAME ) {
+            return;
+        }
+        if (!event.isAvailable) {
+            // Username exists in WordPress.com
+            mUsername = isAvailableUsername;
```

Comment:
It looks like we can rely on `event.value` to give us the user name, so perhaps `isAvailableUsername` isn't necessary?